src/Microsoft.AspNet.Mvc.Core/Rendering/Html/HtmlHelper.cs

```diff
@@ -50,6 +53,7 @@ public class HtmlHelper : IHtmlHelper, ICanHasViewContext
             HtmlEncoder = htmlEncoder;
             UrlEncoder = urlEncoder;
             JavaScriptStringEncoder = javaScriptStringEncoder;
+            _logger = loggerFactory.CreateLogger<HtmlHelper>();
```

Comment:
Please don't start inventing new patterns, Logging has had a ton of discussion and this isn't something we agreed on doing.